exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -19,3 +20,20 @@
  * random (generated in function): 40
  * output: "Sum with value 70 isd left in 30 from number 100"
  */
+
+/**
+ * generate a random number (in between 0-100) and sum it to given number and return a message
+ * @param {number} value - given number
+ * @returns {string} - message
+ */
+function getDistanceMessageFromSumTo100(value) {
+    const random = Math.floor(Math.random() * 100);
+    const sum = value + random;
+    const distance = 100 - sum;
+    if (sum > 100) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (sum > 10THRESHOLD0) {
```

- [ ] Extract number to constant
- [ ] Add empty line between blocks